code/modules/shuttle/shuttle.dm

```diff
@@ -68,7 +68,7 @@
 		return QDEL_HINT_LETMELIVE
 
 /obj/docking_port/has_gravity(turf/T)
-	return FALSE
+	return TRUE
```

Comment:
hm?

Reply:
somewhere during development something made docking ports start drifting away roundstart and it fucked up every shuttle. somewhat hacky fix to that problem but other than that i dont disagree with this change in a vacuum. i added a real fix for this after i traced it to changes in abstract_move()